play/src/front/Components/MapEditor/AreaEditor/AreaPropertiesEditor.svelte

```diff
@@ -641,6 +662,15 @@
                             }}
                             on:change={({ detail }) => onUpdateProperty(property, detail)}
                         />
+                    {:else if property.type === "matrixRoomPropertyData"}
+                        <MatrixRoomPropertyEditor
+                            {property}
+                            on:close={({ detail }) => {
+                                onDeleteProperty(property.id, detail);
+                                if (connection) connection.emitDeleteChatRoomArea(property.matrixRoomId);
```

Comment:
If we delete a WAM file, the Matrix rooms attached won't be deleted. We should probably think about a way to trigger this from the map-storage.